Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -675,6 +682,78 @@ void GameList::OpenSaveFolder()
   QDesktopServices::openUrl(url);
 }
 
+void GameList::OpenGCSaveFolder()
+{
+  const auto game = GetSelectedGame();
+  if (!game)
+    return;
+
+  std::string region;
+
+  switch (game->GetRegion())
+  {
+  case DiscIO::Region::NTSC_J:
+    region = "JAP";
+    break;
+  case DiscIO::Region::NTSC_U:
+    region = "USA";
+    break;
+  case DiscIO::Region::PAL:
+    region = "EUR";
+    break;
+  case DiscIO::Region::Unknown:
+    return;
+  }
+
+  bool found = false;
+
+  for (int i = 0; i < 2; i++)
+  {
+    QUrl url;
+    switch (SConfig::GetInstance().m_EXIDevice[i])
+    {
+    case ExpansionInterface::EXIDEVICE_MEMORYCARDFOLDER:
+    {
+      std::string path = StringFromFormat("%s/%s/%s", File::GetUserPath(D_GCUSER_IDX).c_str(),
+                                          region.c_str(), i == 0 ? "Card A" : "Card B");
```

Comment:
Do we have a central location where this default path is calculated? If so, we should use that one instead of rebuilding it here (especially with those magic strings in there).

Reply:
https://github.com/dolphin-emu/dolphin/blob/c141511c87a8edfa1f273e8fdb13609def89efa8/Source/Core/Core/HW/EXI/EXI_DeviceMemoryCard.cpp#L194-L195

Only the region party, sadly (``SConfig::GetDirectoryForRegion(region) ``).